### PR TITLE
AOM-52: Return and display modules in addition to OWAs on search

### PR DESCRIPTION
**Pull request by mkiterian**
## JIRA TICKET NAME:
[AOM-52 Return and display modules in addition to OWAs on search](https://issues.openmrs.org/browse/AOM-52)
### SUMMARY:
When some one searches, it should return both OWAs and modules that match

### CODE REVIEW

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -390,18 +390,16 @@ export default class ManageApps extends React.Component {
...
-                    downloadUri: res.data['versions'][0].downloadUri,
-                    install: true
-                  };
-                });
+          axios.get(`https://addons.openmrs.org/api/v1//addon/${result.uid}`)
+            .then(res => {
+              this.setState((prevState, props) => {
+                return {
+                  appList: [res.data],
+                  downloadUri: res.data['versions'][0].downloadUri,
```
Comment (jacksono) [inline app/js/components/manageApps/ManageApps.jsx]: Seems you are using 'res' to mean 'response', could you use the full word to make it clearer.

Comment (dkayiwa) [reply app/js/components/manageApps/ManageApps.jsx]: spot on! 👍

**Review by LWanjiru — COMMENTED** [review]
the argument `e`  has been passed and defined in a lot of functions... ie for example catch(e => ,  in the componentDidMount, handleProgress, openModal .. methods, and several other instances on the file. Consider renaming to more descriptive names, if possible. ie catch(error/err => ... or use `event` when it is being used to  handle an event.

File: app/js/components/manageApps/ManageApps.jsx (hunk 1 above)
Comment (mkiterian) [reply app/js/components/manageApps/ManageApps.jsx]: Noted

**Comment by mkiterian** [comment]
This PR was closed because its functionality has been implemented in AOM-47

**Pull request closed without merging**